whylog/tests/tests_front/tests_whylog_factory.py

```diff
@@ -0,0 +1,27 @@
+import os
+import shutil
+from whylog.front.whylog_factory import whylog_factory
+from whylog.log_reader import LogReader
+from whylog.teacher import Teacher
+
+
+class MockClass():
+    def __call__():
+        pass
+
+
+def tests_whylog_factory():
+    try:
+        log_reader, teacher_generator = whylog_factory()
+        teacher = teacher_generator()
+
+        # check if returned object classes
+        assert isinstance(log_reader, LogReader)
+        assert isinstance(teacher, Teacher)
+    except Exception as exc:
+        raise exc
+    finally:
+        path = os.getcwd()
+        config_dir = os.path.join(path, '.whylog_test')
+        if os.path.isdir(config_dir):
+            shutil.rmtree(config_dir)
```

Comment:
Why do we want to delete this file? We didn't created it in this test. It shouldn't exist.